## Jiyuan (Jay) Liu

<!--
**jliu678/jliu678** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on AI x Health
- ðŸŒ± Iâ€™m currently learning AI agents and Transformer engineering
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
Turned complex data into preclinically validated strategies through rigorous biological & mathematical reasoning | Co-Founder, Health x AI | Computational biologist

---
I am a passionate, impact-driven computational biologist who **transforms complex data into preclinically validated cancer prevention and treatment, and drives high-confidence, efficient data-to-decision outcomes**, including hypothesis generation, actionable insights, and preclinical validated strategies.<br>

My (co-)first-author publications demonstrate the impact and outcome: Oncogene(2022 and 2025), EMBO Reports(2023), Nature Communications(2025), two manuscripts in preparation. <br>My computational approach to deliver preclinically validated strategies:<br>&nbsp;&nbsp;&nbsp;&nbsp;(a) seamlessly integrates expertise in computational analysis and bench science;<br>&nbsp;&nbsp;&nbsp;&nbsp;(b) applies robust mathematical and biological reasoning;<br>&nbsp;&nbsp;&nbsp;&nbsp;(c) leverages single-cell and spatial genomics to discover and prioritize cellular and molecular targets.

I represented my university in the National Mathematical Modeling Contest and am highly proficient in developing and customizing computational infrastructure, including:<br>&nbsp;&nbsp;&nbsp;&nbsp;(a) building R packages from scratch;<br>&nbsp;&nbsp;&nbsp;&nbsp;(b) debugging and extending open-source tools in R and Python;<br>&nbsp;&nbsp;&nbsp;&nbsp;(c) developing and optimizing pipelines for translational R&D. 

I have built end-to-end spatial genomics pipelines, covering: sample preparation; sequencing library construction; bioinformatics analysis; publication-ready presentation; preclinical validation.

---

- ðŸ’¬ My accomplished computational biologiy projects are list on [my website](https://jiyuanliu.netlify.app/projects/) as well as:
  - [SeqWins: An R package allowing flexible base trimming and complete Fastq analysis on Windows System as fast and memory-efficient as linux](https://jiyuanliu.netlify.app/post/00a.seqwins/)
  - [Modeling the deposition of infectious agents suggests that aerosolized drugs offer enhanced specificity](https://jiyuanliu.netlify.app/post/00.model_aerosol_depostion/)
  - [Effective RNA Velocity Analysis: Math, Implementation, Benchmark, and Discovery of Meaningful Tumor Trajectories from Real-World Single-Cell Genomics](https://jiyuanliu.netlify.app/project/effective_rna_velcocity/)
  - [Single-Cell Genomic and Clinical Data Analyses Pinpointed Targets and Drove the Preclinically Validated Novel Strategy To Prevent/Treat kidney cancer](https://github.com/jliu678/PRCC-Nature_Communications)
  - [Quantify Un-/Under- Annotated Features (hERVs, Transgenes) from (Single-Cell) Transcriptomics: Math Algorithms, Implementation and Turnkey Workflows](https://jiyuanliu.netlify.app/project/erv_quantification_from_rnaseq/)
  - [Integrated Analysis of Spatial Genomics and scRNAseq Identifies Occult Cervical Cancer Subtypes: Why Current Billion Dollar Screening Programs Fail the Most At-Risk Patients](https://jiyuanliu.netlify.app/project/cervical_cancer/)
  - [Improve UMI Normalization that Determines the Downstream Interpretation of Single-Cell Transcriptomes](https://jiyuanliu.netlify.app/project/improved_scrnaseq_normalization/)
  - [Infection-Driven Uterine Tumorigenesis: Integrated Spatial and Single-Cell Profiling Validated in Patients and Preclinical Models](https://jiyuanliu.netlify.app/project/ut_tumor/)
- ðŸ”­ Iâ€™m currently working on AI x Health and AI agents and Transformer engineering including building nano version of ChatGPT and Deepseek from scratch, LoRA fine tuning from sctratch:
  - I engineered [a high-performance CNN model for histological analysis based on YOLOv8](https://github.com/jliu678/Train-a-high-performance-Yolov8-CNN-model-for-histological-analysis), optimizing model accuracy through data augmentation and implementing Active Learning to combat overfitting
  - I am [certifed AI agents developer](https://www.linkedin.com/posts/jiyuan-liu-7608a42a6_mindstudio-aiagents-automation-activity-7370679693399715840-Si3B?utm_source=share&utm_medium=member_desktop&rcm=ACoAAEnQiTgBVixA2fAJogfjA8v6P8FFk_wBXXo) 
  - My Hackthon project: [An multimodal agentic EHR co-pilot](https://github.com/jliu678/AI-x-Bio-Hackathon-Evolved-Boston-2025)

- ðŸ‘¯ Tools I collobratively contributed to
  - [The rseqR package: perform differential analysis of RNA-seq data](https://github.com/jliu678/rseqR_myPRapproved)(my PR got [approved](https://github.com/anilchalisey/rseqR/pull/2#pullrequestreview-1178152122) and merged with other improvements)
  - [cellcuratoR: interactive single-cell gene expression visualizations from Seurat](https://github.com/jliu678/cellcuratoR_myPRmerged)(my PR got [merged](https://github.com/jliu678/cellcuratoR_myPRmerged/commits/master/))
  - [My edition of stereopy: a fundamental and comprehensive tool for mining and visualization based on spatial transcriptomics data](https://github.com/jliu678/stereopy_jliu678-edits)(I fixed bugs in h5ad2rds.R and enabled successful conversion of spatial dataset from python h5ad to R seurat object)
  - [My edition of sctransform: a R package normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression](https://github.com/jliu678/sctransform_jliu678-edits)(I add new functions allow the theta_given to be effetive together with vst.flavor = "v2", thus to use the stabler glmgampoi (implemented in sctransform v2) to run the analytic pearson residual. Check [my blog](https://jiyuanliu.netlify.app/project/improved_scrnaseq_normalization/) to learn the logic flow of the original sctransform() and the improved UMI normalization model.)
  - [My edition of msigdbr: MSigDB gene sets typically used with the Gene Set Enrichment Analysis (GSEA)](https://github.com/jliu678/msigdbr_jliu678-edits)(I updated the source genesets, and enabled using mouse genesets directly from Msigdb rather than converted from human genesets based on orthologs. My PR was not approved at that time, but the newer version acutally adopted my idea.)

- ðŸ“– Love to share my [computational biology blogs](https://jiyuanliu.netlify.app/post/), where I am writting to fill gaps I couldnâ€™t find clearly addressed elsewhere, offering original insights and mathematical reasoning behind bioinformatics tools and large language models and more.
